### DISCUSSION TITLE
New! Try out the Amazon Ads API integration dashboard

### DISCUSSION THREAD

**Discussion by elizafarley** [amzn/ads-advanced-tools-docs#223] [Announcements]
At the beginning of February, Amazon Ads introduced a new, personalized experience for developers using the Amazon Ads API. With this release, developers can now log in to the [advanced tools center](https://advertising.amazon.com/API/docs/en-us) to view key metrics and trends related to their Amazon Ads API integration.

The logged-in experience displays information about your Login with Amazon application and includes an interactive dashboard showcasing API activity. The integration dashboard can help you monitor your integration health, explore trends, and identify inefficiencies in your call patterns. Available breakdowns include calls by response code, product, marketplace, API resource, and hourly call volume.

For more details and use cases of the new integration dashboard, see our [Amazon Ads integration dashboard guide](https://m.media-amazon.com/images/S/aapn-assets-prod/86920bb8-a600-4c19-a00a-c27ea24b4280).

**Accessing the dashboard**

Access the new experience directly from the Amazon Ads advanced tools center by signing in using your developer.amazon.com account credentials:

<img width="1438" alt="Screenshot 2024-02-22 at 4 38 39 PM" src="https://github.com/amzn/ads-advanced-tools-docs/assets/6935415/9f3795b8-354b-4694-99bf-0888dab2e09f">

Once logged in, you can view the Login with Amazon applications associated with your Amazon Developer account. Select an application that has access to the Amazon Ads API to view the integration dashboard:

![Dashboard page](https://github.com/amzn/ads-advanced-tools-docs/assets/6935415/f3a6bb20-56f3-4049-b85e-6f295d6ddb7e)

**Feedback?**

Have you logged in to see your integration dashboard yet? Let us know what you think and if you have any feature requests!

**Comment by AdsAPIPM** [comment]
Very cool

**Comment by Sourabh-jana** [comment]
Very insight full 👌😊

**Comment by PabloTetris** [comment]
Amazing!
It would be awesome if we could see the average time to generate reports, especially for SPAds. 
We see giant degradation in time on big days like Black Friday, for example, and it is important to be able to track that easily.

**Reply by elizafarley** [reply]
@PabloTetris thanks for sharing this feedback and your use case! I've passed this info on to the product team. Stay tuned for any new features in our [release notes](https://advertising.amazon.com/API/docs/en-us/release-notes/index).
